scripts/scenarios/cloud6/cloud6-2nodes-default.yml

```diff
@@ -0,0 +1,131 @@
+---
+proposals:
+- barclamp: database
+  attributes:
+  deployment:
+    elements:
+      database-server:
+      - @@controller@@
+- barclamp: rabbitmq
+  attributes:
+  deployment:
+    elements:
+      rabbitmq-server:
+      - @@controller@@
+- barclamp: keystone
+  attributes:
+  deployment:
+    elements:
+      keystone-server:
+      - @@controller@@
+- barclamp: swift
+  attributes:
+    zones: 1
+    keystone_delay_auth_decision: true
+    allow_versions: true
```

Comment:
Reviewing again: I'd feel more comfortable if we could explicitly set some more attributes here, even if they're already set correctly in the template:
https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crowbarsetup.sh#L2247

(these are required for tempest tests)